src/components/WheelPicker/index.tsx

```diff
@@ -351,7 +351,7 @@ const WheelPicker = <T extends WheelPickerItemValue>(props: WheelPickerProps<T>)
     <View testID={testID} bg-$backgroundDefault style={style}>
       {separators}
       <View row centerH>
-        <View flexG>
+        <GestureHandlerRootView style={{flexGrow: 1}}>
```

Comment:
avoid passing inline style, let's move to stylesheet

Reply:
Done